src/mame/drivers/h01x.cpp

```diff
@@ -0,0 +1,435 @@
+// license:BSD-3-Clause
```

Comment:
This file lacks a copyright holder.  Since we have no legal entity or contributor license agreement, all files must list copyright holders as well as licenses.